### PR TITLE
Update ssrbool.v

### PR DESCRIPTION
**Pull request by jouvelot**
Adding comments on negPf and idPn.

Fixes / closes #????

- [ ] Added / updated **test-suite**.

- [ ] Added **changelog**.
- [ ] Added / updated **documentation**.
  
  - [ ] Documented any new / changed **user messages**.
  - [ ] Updated **documented syntax** by running `make doc_gram_rsts`.

- [ ] Opened **overlay** pull requests.

<!-- Pointers to relevant developer documentation:

Contributing guide: https://github.com/coq/coq/blob/master/CONTRIBUTING.md

Test-suite: https://github.com/coq/coq/blob/master/test-suite/README.md

Changelog: https://github.com/coq/coq/blob/master/doc/changelog/README.md

Building the doc: https://github.com/coq/coq/blob/master/doc/README.md
Sphinx: https://github.com/coq/coq/blob/master/doc/sphinx/README.rst
doc_gram: https://github.com/coq/coq/blob/master/doc/tools/docgram/README.md

Overlays: https://github.com/coq/coq/blob/master/dev/ci/user-overlays/README.md

### CODE REVIEW

File: theories/ssr/ssrbool.v [hunk 1]
```diff
@@ -922,6 +923,7 @@ Proof. by case b1; constructor; auto. Qed.
 Lemma negPn : reflect b1 (~~ ~~ b1).
 Proof. by case b1; constructor. Qed.
 
+(* Right-to-left reflection, no-op otherwise. *)
```
[the remark this answers is not in the document]
Comment (jouvelot) [reply theories/ssr/ssrbool.v:926]: If we go for an extended comment like this, shouldn't I also do the same for idPn?

[the remark this answers is not in the document]
Comment (proux01) [reply theories/ssr/ssrbool.v:926]: If you want but I'm not sure it's really needed. I don't see much risk of users being surprised by the behavior of `idPn`.

[the remark this answers is not in the document]
Comment (jouvelot) [reply theories/ssr/ssrbool.v:926]: No problem. Let's leave it at that, then :)

**Review by proux01 — APPROVED** [review]

**Comment by proux01** [comment]
@coqbot merge now

**Pull request merged**